utilities/install-hooks.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+chmod +x utilities/hooks/*
```

Comment:
suggestion: can we somehow run this shell script from main build.sh? in that case we will not have to do anything manually

Reply:
it is doable but, `build.sh` would not be the best place.